cost-analyzer/values.yaml

```diff
@@ -217,8 +217,20 @@ global:
     # iam.amazonaws.com/role: role-arn
   additionalLabels: {}
 
-  containerSecuritycontext: {}
-    # readOnlyRootFilesystem: true
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+    fsGroup: 65534
+    runAsGroup: 65534
+    runAsUser: 65534
```

Comment:
Any reason we've chosen `65534` here? Additionally, have we confirmed that all the containers in our chart can successfully run using a different user, with a `readonly` filesystem, and with all its capabilities dropped?

Reply:
yes, I did. 
the user works well for openshift and non-openshift tests I ran.

Reply:
Ah! According to stackexchange the significance of `65534` is that Linux reserves this for the `nobody` user/group. https://unix.stackexchange.com/questions/720557/what-is-the-id-of-nobody-user-and-nogroup-group